test/floatapprox.jl

```diff
@@ -6,19 +6,19 @@
 @test 4.000000002 ≉ 4.00300002
 
 # Other tolerance levels
-@test isapprox(4.32, 4.3; rtol=0.1, atol=0.01)
-@test isapprox(1.001, 1.002; rtol=0.001, atol=0.0001)
-@test !isapprox(4.5, 4.9; rtol=0.001, atol=0.001)
+@test 4.32 ≈ 4.3 rtol=0.1 atol=0.01
+@test 1.001 ≈ 1.002 rtol=0.001 atol=0.0001
+@test 4.5 ≉ 4.9 rtol=0.001 atol=0.001
```

Comment:
I just added it recently: https://github.com/JuliaLang/julia/pull/19901/commits/9c58797498980e283cbe7ed59ae58353fbc24c7b. That commit also adds the following documentation:

>     @test ex
>     @test f(args...) key=val ...
> Tests that the expression `ex` evaluates to `true`. Returns a `Pass` `Result` if it does, a `Fail` `Result` if it is `false`, and an `Error` `Result` if it could not be evaluated. The `@test f(args...) key=val...` form is equivalent to writing `@test f(args..., key=val...)` which can be useful when the expression is a call using infix syntax such as approximate comparisons:
> 
>     @test a ≈ b atol=ε
> 
> This is equivalent to the uglier test `@test ≈(a, b, atol=ε)`. It is an error to supply more than one expression unless the first is a call expression and the rest are assignments (`k=v`).